src/main/java/com/cmtelecom/text/sdk/utils/HttpHelper.java

```diff
@@ -19,13 +19,11 @@ private static String sendRequest( String method, String url, String data ) {
             
             conn.setRequestProperty( "Content-Type", "application/json" );
             
-            
             if ( data != null ) {
-                // Send request
-                           
+                // Send request    
                 conn.setDoOutput( true );
                 try ( DataOutputStream wr = new DataOutputStream( conn.getOutputStream() ) ) {
-                    wr.writeBytes( data );
+                    wr.write( data.getBytes(StandardCharsets.UTF_8) );
```

Comment:
Did you test thoroughly with different UTF8/16 and emoji's for example here?
I think this is fine, but we are forcing to UTF8 here then.

Reply:
emoji are not supported at the moment, there is an different ticket for that ⚡